voucher/src/main/java/com/zerozae/voucher/domain/voucher/PercentDiscountVoucher.java

```diff
@@ -0,0 +1,25 @@
+package com.zerozae.voucher.domain.voucher;
+
+import lombok.Getter;
+
+import java.util.UUID;
+
+@Getter
+public class PercentDiscountVoucher implements Voucher{
```

Comment:
전반적으로 포매팅 확인 부탁드립니다~
```suggestion
public class PercentDiscountVoucher implements Voucher {
```

Reply:
확인하고 반영 하겠습니다 !